cloud/google/create_new_cluster.sh

```diff
@@ -45,7 +45,7 @@ function PopulateEtcdForLog() {
   gcloud compute ssh ${ETCD_MACHINES[1]} \
       --zone ${ETCD_ZONES[1]} \
       --command "\
-    sudo docker run gcr.io/${PROJECT}/super_duper:test \
+    sudo docker run gcr.io/${PROJECT}/ct-log:test \
```

Comment:
While you're here, could you pop a TODO in there for not bumping the `test` label every time, but rather create new versioned labels each time?